api/src/main/java/com/codeforcommunity/dto/map/SiteFeatureProperties.java

```diff
@@ -0,0 +1,62 @@
+package com.codeforcommunity.dto.map;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.sql.Timestamp;
+
+public class SiteFeatureProperties {
+  Integer id;
+  Boolean treePresent;
+  Double diameter;
+  String species;
+  Timestamp updatedAt;
+  String updatedBy; // username
```

Comment:
Stray comment?

Reply:
I figured it would help to clarify what updatedBy was referring to